### PR TITLE
Column ordering

### PR DESCRIPTION
**Pull request by phlax**
this PR will reorder columns in table views to emphasise better recent activity

- move last activity column to the left
- move updated column last
- make last updated admin only
- ~~make default ordering by last activity~~ <-- gonna do this separately

### CODE REVIEW

File: pootle/apps/virtualfolder/views.py [hunk 1]
```diff
@@ -114,20 +114,26 @@ def get_vfolder_units(request, **kwargs):
...
 
     @property
     def table_fields(self):
-        return [
-            'name', 'priority', 'progress', 'total',
-            'need-translation', 'suggestions', 'critical',
-            'last-updated', 'activity']
+        fields = self._table_fields
+        if self.has_admin_access:
+            fields = fields + ('last-updated', )
```
Comment (unho) [inline pootle/apps/virtualfolder/views.py]: Nit:

```python
            fields += ('last-updated', )
```

File: pootle/apps/pootle_app/panels.py [hunk 2]
```diff
@@ -19,9 +19,17 @@
...
+    _table_fields = (
+        'name', 'progress', 'activity',
+        'total', 'need-translation',
+        'suggestions', 'critical')
+
+    @property
+    def table_fields(self):
+        fields = self._table_fields
+        if self.view.has_admin_access:
+            fields = fields + ('last-updated', )
```
Comment (unho) [inline pootle/apps/pootle_app/panels.py]: Nit again:

```python
            fields += ('last-updated', )
```

File: pootle/apps/pootle_app/panels.py [hunk 3]
```diff
@@ -19,9 +19,17 @@
 
 class ChildrenPanel(TablePanel):
     panel_name = "children"
-    table_fields = [
-        'name', 'progress', 'total', 'need-translation',
-        'suggestions', 'critical', 'last-updated', 'activity']
+    _table_fields = (
+        'name', 'progress', 'activity',
+        'total', 'need-translation',
+        'suggestions', 'critical')
```
Comment (unho) [inline pootle/apps/pootle_app/panels.py]: Wrong order? I think it should be:

Nit:

```python
    _table_fields = (
        'name', 'progress', 'activity',
        'critical', 'suggestions', 'need-translation', 'total', )
```

Comment (unho) [inline pootle/apps/pootle_app/panels.py]: Just noticed it doesn't matter the order here, but only when rendering.

Comment (phlax) [inline pootle/apps/pootle_app/panels.py]: this isnt used for ordering

**Comment by unho** [comment]
Update "Update table include template" commit message to also tell that "Total" has been moved as well.

**Comment by unho** [comment]
Reviewed and tested. Other than nits, gtm.

**Pull request merged**